### PR TITLE
Introduce Jobs page

### PR DESCRIPTION
**Pull request by barinali**

### CODE REVIEW

**Comment by shadzik** [comment]
LGTM, as far as the code goes. As for the content you probably need another reviewer.

**Review by shadzik — APPROVED** [review]

File: posts/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h2>Blog</h2>
     <article>
       <header>
         <div class="date-orange">
-        <time datetime="2017-02-14">{{ post.date | date: site.date_format }}</time>
+        <time datetime="{{ post.date | date: '%Y-%m-%d' }}">{{ post.date | date: site.date_format }}</time>
```
Comment (shadzik) [inline posts/index.html:16]: Thanks for this @barinali

Comment (barinali) [reply posts/index.html:16]: You are welcome. I realized it while getting influenced by the /posts page and thought I'd fix it while I'm at it. :)

File: _jobs/front-end-engineer.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+We appreciate the following **general capabilities**:
+
+*   fluent in English
+*   agile practitioner
+*   team-player in remote/distributed setting
+*   savvy in video-conference-meetings
+*   eager to learn new technologies
+
+The recruitment process is mostly based on two screenings: one with The WorkinGroup and the second with client.
```
Comment (stefanwascoding) [inline _jobs/front-end-engineer.md]: @barinali That last sentence needs to go.

**Review by stefanwascoding — APPROVED** [review]

**Review by stefanwascoding — APPROVED** [review]

**Pull request merged**